### DISCUSSION TITLE
Fixing broken images in *.d.ts JSDoc blocks & related Intellsense pop-ups

### DISCUSSION THREAD

**Discussion by dangrussell** [ReactiveX/rxjs#6973] [Report issues other than bug]
Declaration files contain relative references to image files that display as broken images in VSCode,

For example, the JSDoc block for [toArray ](https://github.com/ReactiveX/rxjs/blob/master/src/internal/operators/toArray.ts), contains this line:
`* ![](toArray.png)`

If we used an absolute URL, the image would then be displayed correctly in VSCode's Intellisense pop-up:
`* ![](https://rxjs.dev/assets/images/marble-diagrams/toArray.png)`

The .png files aren't included in the npm packages, so they would never be available locally. I'm not sure if these declaration files are being consumed elsewhere where the relative paths would make sense. (Are these files the source for the docs website? They don't seem to be...)

Fixing up these URLs would also be a good chance to add some alt text to the these images, which is missing currently even with the relative path, i.e.:
`* ![marble diagram for toArray operator](https://rxjs.dev/assets/images/marble-diagrams/toArray.png)`

Alternately, these image links could be removed entirely.

**Comment by tmair** [comment]
There is already an issue adressing this: https://github.com/ReactiveX/rxjs/issues/5978

Fixing the issue is not totally trivial, as the JSDoc is also used to generate the API documentation on rxjs.dev
